### PR TITLE
CB-6258 Use Launch Templates in CloudFormation templates

### PR DESCRIPTION
**Pull request by Bajzathd**

### CODE REVIEW

File: cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl [hunk 1]
```diff
@@ -120,7 +120,10 @@
         <#else>
         "VPCZoneIdentifier" : [{ "Ref" : "SubnetId" }],
         </#if>
-        "LaunchConfigurationName" : { "Ref" : "AmbariNodeLaunchConfig${group.groupName?replace('_', '')}" },
+        "LaunchTemplate" : {
+          "LaunchTemplateId": { "Ref" : "AmbariNodeLaunchTemplate${group.groupName?replace('_', '')}" },
```
Comment (doktoric) [inline cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl]: AmbariNodeLaunchTemplate -> CDPNodeLaunchTemplate

File: cloud-aws/src/main/resources/templates/aws-cf-stack.ftl [hunk 2]
```diff
@@ -312,7 +312,10 @@
         "VPCZoneIdentifier" : [{ "Ref" : "SubnetId" }],
         </#if>
         </#if>
-        "LaunchConfigurationName" : { "Ref" : "AmbariNodeLaunchConfig${group.groupName?replace('_', '')}" },
+        "LaunchTemplate" : {
+          "LaunchTemplateId": { "Ref" : "AmbariNodeLaunchTemplate${group.groupName?replace('_', '')}" },
```
Comment (doktoric) [inline cloud-aws/src/main/resources/templates/aws-cf-stack.ftl]: CDPNodeLaunchTemplate

File: cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl (hunk 1 above)
Comment (bergerdenes) [reply cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl]: or rather ClusterManagerNodeLaunchTemplate

Comment (doktoric) [reply cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl]: Both are fine but cf template has a limit on the length this is why i think shorter is better

Comment (Bajzathd) [reply cloud-aws/src/main/resources/templates/aws-cf-stack-freeipa.ftl]: Renamed it to ClusterManagerNodeLaunchTemplate, ran just fine

**Pull request merged**